modules/ossm-performing-inplace-update.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: Procedure
+[id="performing-inplace-updtate"]
+= Performing InPlace update
+:context: ossm-performing-inplace-update
+
+When updating {istio} using the `InPlace` strategy, you can only increment by one minor version at a time. If you want to update by more than one minor version, then you must increment the version and restart the workloads after each update. This ensures that the sidecar version is compatible with the control plane version. After all the workloads are restarted the update process is complete.
+
```

Comment:
@luksa can you think of any prerequisites we should add for this procedure? Does the user performing the upgrade have to be an admin?

Reply:
Yes, the user has to be a cluster admin to create & update the Istio custom resource.

Can't think of any other prerequisites except that the control plane has to be installed, but I guess that's obvious.

Reply:
updated